ikarus/finiteelements/mechanics/membranestrains.hh

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#pragma once
+#include <ranges>
+
+#include <dune/common/fvector.hh>
+#include <dune/common/overloadset.hh>
+#include <dune/localfefunctions/impl/standardLocalFunction.hh>
+
+#include <Eigen/Core>
+namespace Ikarus {
+
+  struct DefaultMembraneStrain {

```

Comment:
here a concept can be added such that the user can add different membrane strain computations adhering to a particular concept?